joomla/joomla-cms
# Option to disable save article hits

Pull Request for Issue #22406 .

### Summary of Changes
Option for Disable Hits counter increment


### Testing Instructions
1. Open an article and check if the hits do increase
2. Go to content options and Set "Save Hits" to "NO"
3. Open an article and check if the hits do not increase


### Expected result
If disabled, article hits do not increase


### Actual result
article hits do not increase always


### Documentation Changes Required
no?
